README.md

```diff
@@ -13,6 +13,24 @@ This extension modifies the Saved Replies for issues or pull requests on any rep
 
 Get it on [Chrome Web Store > Refined Saved Replies](https://chrome.google.com/webstore/detail/refined-saved-replies/ngcinicnlicdndmpcfjjifononfcceih)! ✨
 
+Install it from the [Firefox Add-ons > Refined Saved Replies](#TBD) page! 🦊
+
+After installing the extension in Firefox, you'll need to authorize the extension to run on [https://github.com](https://github.com).
```

Comment:
[Proofreading] This line looks like it belongs inside the _Firefox Configuration_ section, no?